pkg/cidata/cidata.TEMPLATE.d/boot/05-rosetta-volume.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+if [ ! -d /tmp/rosetta-share ]; then
+  mkdir -p /mnt/lima-rosetta
+  # Cannot find a way to check that the virtiofs "device" is present, so just
+  # Try mounting it and hope for the best. If it fails, then it probably isn't
+  # a mac, or the virtio volume isn't being provided to the guest.
+  mount -t virtiofs vz-rosetta /mnt/lima-rosetta || true
```

Comment:
Good thought, I'll see how difficult that is. I'm not sure if mounts works for anything besides bind-mounts, but if it does, then that would work.

Reply:
Ah !! It won't work i think.
I missed a case, it will not work when we have mix of mountType. Using mountType: reversessh-fs and wanting to use rosetta.
This will not work as all the y.mounts are done based on mountType.

Reply:
We could add another mountType for this potentially. Not sure if it would be useful outside of this use-case yet though.

Reply:
I ended up continuing to use cidata, but we can revisit the approach later if needed.